tests/schema_registry/test_avro_serializer.py

```diff
@@ -34,7 +34,7 @@ def test_avro_serializer_config_auto_register_schemas():
     """
     conf = {'url': TEST_URL}
     test_client = SchemaRegistryClient(conf)
-    test_serializer = AvroSerializer(test_client, 'string',
+    test_serializer = AvroSerializer(test_client, '"string"',
```

Comment:
This seems like a breaking change. Let's keep the old behaviour.

Reply:
i'm quite sure this is strictly a bug fix, not a breaking change, see previous comment.